charts/cloud-controls/templates/club-pols.yaml

```diff
@@ -0,0 +1,17 @@
+{{- define "parenttemplate" }}
```

Comment:
In this case, how do we provide policy specific parameters? Example, if the policy checks that images are pulled from the allowed list of registries, and user has to provide this as input.